lib/ukio/bookings/handlers/booking_creator.ex

```diff
@@ -1,6 +1,7 @@
 defmodule Ukio.Bookings.Handlers.BookingCreator do
   alias Ukio.Apartments
   alias Ukio.Apartments.Handlers.AvailabilityChecker
+  @utilities_fee_multiplier %{"mars" => 150}
```

Comment:
If we were to have a new `Market` model, this information would belong to its schema.
For now, just using a Map, where the keys would be the different markets and the values the utilities fee multiplier that we need to use.
It's easy to expand to other markets by just adding new key-value pairs